airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/commands.py

```diff
@@ -124,6 +125,17 @@ async def test(
 
     if ctx.obj["selected_connectors_with_modified_files"]:
         update_global_commit_status_check_for_tests(ctx.obj, "pending")
+        if any([connector.support_level == "certified" for connector in ctx.obj["selected_connectors"]]):
```

Comment:
It might be worth adding a comment about why we're emitting this status check even if the pipeline is not running regression tests. (It could be added the `hacks.py`)

Reply:
Done.